src/qp/quant_pdf.py

```diff
@@ -207,139 +266,3 @@ def add_mappings(cls):
                                           convert_data=dict(quants=QUANTS), test_xvals=TEST_XVALS))
 
 add_class(quant_gen)
-
-
-
-
-class quant_piecewise_gen(Pdf_rows_gen):
```

Comment:
Completely removed this class because it is now superfluous. It was originally created so that an alternative pdf reconstruction algorithm could be implemented. Now that the pdf reconstruction algorithms are separated into their own classes and selectable at run time, we no longer need this class.